packages/editor/src/pages/Editor/store/gists/sagas.ts

```diff
@@ -14,6 +14,7 @@ import { createSolutionSaga } from '../solutions/sagas';
 import { checkForUnsupportedAPIsIfRelevant } from './utilities';
 
 export default function* gistsWatcher() {
+  yield takeEvery(getType(host.change), fetchAllGistMetadataSaga);
```

Comment:
This needed to be added due to the fact that the functionality used to exist in the local host sagas, but doesn't in script-lab-core.'s host sagas.